### PR TITLE
Add testing episode

### PR DESCRIPTION
**Pull request by PabRod**

### CODE REVIEW

File: _episodes/07-testing.md [hunk 1]
```diff
@@ -0,0 +1,61 @@
...
+You already know the very basics of testing!
+
+The main lesson we'll get out of this episode is that those kind of tests are extremely valuable.
+So much, that they deserve more than just be typed on the console and be forgotten forever.
+Tests deserve to be saved, and even more, rerun often.
+
+## Why shall I store my tests?
+
+The vast majority of people needs to try their scripts and functions in order to make sure they work fine.
+And we do it.
```
Comment (liekelotte) [inline _episodes/07-testing.md:28]: I'm not sure entirely what is meant by "it" here

File: _episodes/07-testing.md [hunk 2]
```diff
@@ -0,0 +1,61 @@
...
+What is the added value of storing those tests?
+> ## What is the added value of storing my tests?
+> Storing tests in the form of a script takes a bit more time and effort than just running them interactively in the console.
+>
```
Comment (liekelotte) [inline _episodes/07-testing.md:32]: Should the participants not know what a test is before they can answer this question?

**Review by liekelotte — APPROVED** [review]
Looks like a good start!!

File: _episodes/07-testing.md (hunk 1 above)
Comment (PabRod) [reply _episodes/07-testing.md:28]: True. Fixed it. Thanks!

File: _episodes/07-testing.md (hunk 2 above)
Comment (PabRod) [reply _episodes/07-testing.md:32]: That's a good one. I am trying to let the reader naturally drift between the intuitive, somehow loose concept of _"test"_ into the technical one of _"unit test with testthat"_. I'll think about it.

**Comment by PabRod** [comment]
> Looks like a good start!!

Thanks. Indeed, I realized I assigned you to review too early 😅. Sorry for that

**Pull request merged**